auslib/web/admin/views/mapper.py

```diff
@@ -0,0 +1,76 @@
+from auslib.web.admin.views.csrf import CSRFView
+from auslib.web.admin.views.rules import RulesAPIView
+from auslib.web.admin.views.permissions import UsersView
+from jsonschema import draft4_format_checker
+from jsonschema.compat import str_types
+from wtforms.validators import ValidationError
+from auslib.util.comparison import get_op
+from auslib.util.versions import MozillaVersion
+import operator
+import logging
+
+log = logging.getLogger(__name__)
+
+
+def csrf_get():
+    return CSRFView().get()
+
+
+def rules_get():
+    return RulesAPIView().get()
+
+
+def rules_post():
+    return RulesAPIView().post()
+
+
+def users_get():
+    return UsersView().get()
+
+
+@draft4_format_checker.checks(format="buildID", raises=ValidationError)
+def operator_validator(field_value):
+    log.debug('starting in operator_validator: buildID is %s' % field_value)
+    if not isinstance(field_value, str_types):
+        return True
+    # empty input is fine
+    if field_value is None or field_value == '':
+        return True
```

Comment:
It's unfortunate that the UI sends an empty string (it should really send null for empty fields) - we should fix that at some future point. I'm still surprised that you need to handle None here though. Does the validator even get called when the value is None/null? Typically validators aren't called when the data isn't present, but maybe swagger/connexion work differently.